src/renderer.rs

```diff
@@ -165,6 +168,17 @@ pub fn page(
                             }
                         }
                     }
+            @if readme.is_some() {
+            div {
+                h3 { (readme.as_ref().unwrap().file_name().unwrap()
+                  .to_string_lossy().to_string()) }
+                (PreEscaped
+                 (markdown_to_html(
+                 &std::fs::read_to_string(readme.unwrap())
+                     .unwrap_or_else(|_| "Cannot read File.".to_string()),
```

Comment:
I think we should probably not do heavy IO work during template-time. Can you move the file reading into `listing.rs` as well?

Reply:
I did try that, then we'd have to pass filename (could be variation on capitalization) and the content separately into 2 variables and I didn't want to do that, we can remove the filename from the render but I think it's better to be consistent with github as that's where we got the inspiration from.

Reply:
It's done now.